ResearchKit/ActiveTasks/ORKRangeOfMotionStep.h

```diff
@@ -0,0 +1,28 @@
+//
+//  ORKRangeOfMotionStep.h
+//  ResearchKit
+//
+//  Created by Darren Levy on 8/20/16.
+//  Copyright © 2016 researchkit.org. All rights reserved.
+//
```

Comment:
Update headers to match RK legal headers. Refer to any of the other RK header files and copy/paste the legal header.

Replace Copyright Apple with your name. Look at other contributions from the past for reference on the headers.